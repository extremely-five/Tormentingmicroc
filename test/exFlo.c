void main(){
    float a;
    a = 3.0;
   

}
