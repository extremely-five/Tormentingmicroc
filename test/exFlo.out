24 19 0 5 25 15 1 21 0